CMakeLists.txt

```diff
@@ -25,6 +25,16 @@ cmake_minimum_required(VERSION 3.1.3)
 project(LibTaskForce LANGUAGES CXX)
 set(CMAKE_CXX_EXTENSIONS False CACHE BOOL "Enable/Disable compiler-specific C++ extensions")
 set(CMAKE_CXX_STANDARD 14 CACHE STRING "Which C++ standard to use")
+set(LIBTASKFORCE_BUILD_SHARED FALSE CACHE BOOL "Build a shared library, rather than a static library")
```

Comment:
There's a very good reason for that - I didn't know about that variable. That might clean up bphash/bpprint also